mico-admin/src/app/dialogs/service-picker/service-picker.component.html

```diff
@@ -8,18 +8,23 @@ <h2 mat-dialog-title>Pick a Service</h2>
     <table mat-table [dataSource]="dataSource" style="width: 100%">
 
         <!-- Checkbox Column -->
-        <ng-container matColumnDef="select" class="pr2">
-            <th mat-header-cell *matHeaderCellDef>
-                <mat-checkbox (change)="$event ? masterToggle() : null" [checked]="selection.hasValue() && isAllSelected()"
-                    [indeterminate]="selection.hasValue() && !isAllSelected()">
-                </mat-checkbox>
-            </th>
-            <td mat-cell *matCellDef="let row">
-                <mat-checkbox (click)="$event.stopPropagation()" (change)="$event ? selection.toggle(row) : null"
-                    [checked]="selection.isSelected(row)">
-                </mat-checkbox>
-            </td>
-        </ng-container>
+        <div *ngIf="choiceModel==1">
+            <!-- 1: multi
+            insert checkboxes
+            -->
+            <ng-container matColumnDef="select" class="pr2">
```

Comment:
```suggestion
            <ng-container *ngIf="choiceModel==1" matColumnDef="select" class="pr2">
```

Don't add new divs for ngIf if possible.